cdc/owner/owner.go

```diff
@@ -173,13 +175,21 @@ func (o *ownerImpl) Tick(stdCtx context.Context, rawState orchestrator.ReactorSt
 				upstreamInfo := state.Upstreams[changefeedState.Info.UpstreamID]
 				up = o.upstreamManager.AddUpstream(upstreamInfo)
 			}
-			cfReactor = o.newChangefeed(changefeedID, changefeedState, up, o.cfg)
+			cfReactor = o.newChangefeed(changefeedID, changefeedState.Info, changefeedState.Status,
+				newFeedStateManager(up, changefeedState),
+				up, o.cfg)
 			o.changefeeds[changefeedID] = cfReactor
 		}
 		ctx = cdcContext.WithChangefeedVars(ctx, &cdcContext.ChangefeedVars{
 			ID: changefeedID,
 		})
-		cfReactor.Tick(ctx, o.getChangefeedCaptures(changefeedState, state))
+		changefeedState.CheckCaptureAlive(ctx.GlobalVars().CaptureInfo.ID)
```

Comment:
Why this is needed?

Reply:
just keep the old logic 
checks if the capture is alive, if the capture offline,  the etcd worker will exit and throw the ErrLeaseExpired error.